common/stacks/src/main/scala/weco/api/stacks/services/ItemLookup.scala

```diff
@@ -17,16 +17,12 @@ import scala.concurrent.{ExecutionContext, Future}
 class ItemLookup(elasticsearchService: ElasticsearchService)(
   implicit ec: ExecutionContext
 ) {
-
-  /** Returns an Item that corresponds to this canonical ID.
-    *
-    * Note: the output from this function may not be consistent, even against
-    * a fixed API index.  Because of merging logic, some item IDs have varying
-    * representations on different works.
+  /** Returns the SourceIdentifier of the item that corresponds to this
+    * canonical ID.
     *
     */
   def byCanonicalId(itemId: CanonicalId)(index: Index)
```

Comment:
Can't an item looked up this way have multiple `SourceIdentifier`s?

Reply:
Ah, so I meant this to be "the SourceIdentifier that corresponds to the CanonicalId" (i.e. what you'd get in the ID minter table) – but hmm, yeah, maybe it should be a list.

Reply:
Now I think about this – I think I do want the canonical SourceIdentifier for now; the other lists can get unwieldy.

Reply:
More specifically: once you start looking at multiple identifiers, you multiply the potential error states with duplicates/mismatches/unrelated identifiers. I think we could support it if we wanted to, but there's no compelling reason to look at anything but the first.